### DISCUSSION TITLE
Would it be possible to add auto labeling support for Detectron or Yolo Models?

### DISCUSSION THREAD

**Discussion by ajenez** [ryouchinsa/Rectlabel-support#243] [Ideas]
My team would find it very helpful to be able to use either our created Detectron or Yolo models for autolabeling. Is it possible to add those models for Autolabeling?

**Comment by ryouchinsa** [comment]
Thanks for writing the issue.
For YOLOv5/v8 models, you can convert them to Core ML models and auto label using the model on RectLabel.

Using the script [general_json2yolo.py](https://github.com/ryouchinsa/Rectlabel-support/blob/master/general_json2yolo.py), you can convert the RLE mask with holes to the YOLO segmentation format.

Related issue.
https://github.com/ultralytics/JSON2YOLO/issues/38
